### PR TITLE
smoke

### PR DESCRIPTION
**Pull request by GH-integration**

### CODE REVIEW

**Comment by iluxa-com** [comment]
test

**Comment by iluxa-c0m** [comment]
sdaf

**Comment by iluxa-com** [comment]
review comment

**Comment by iluxa-com** [comment]
new text111

**Comment by iluxa-com** [comment]
up review comment

**Comment by iluxa-com** [comment]
GH review comment

File: src/main/java/com/bangsapabbi/helloworld/HelloWorld.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 public class HelloWorld {
     public static void main(String[] args) {
-    World world = new World();    
+        World world = new World();    //
```
Comment (iluxa-com) [inline src/main/java/com/bangsapabbi/helloworld/HelloWorld.java:5]: inline comment in Upsource

Comment (iluxa-com) [reply src/main/java/com/bangsapabbi/helloworld/HelloWorld.java:5]: reply from GH

Comment (iluxa-c0m) [reply src/main/java/com/bangsapabbi/helloworld/HelloWorld.java:5]: and me!?

**Pull request merged**